src/tests/ftest/object/create_many_dkeys.py

```diff
@@ -4,15 +4,15 @@
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 '''
-from __future__ import print_function
+
 
 import sys
 import ctypes
 import avocado
 
 from apricot import TestWithServers
 from pydaos.raw import DaosContainer, IORequest, DaosApiError
-
+from general_utils import create_string_buffer
```

Comment:
(pylint-syntax-error) Cannot import 'general_utils' due to syntax error 'invalid syntax (<unknown>, line 888)'